iOS/G3MiOSSDK/Commons/Rendererers/TileLoDTester.hpp

```diff
@@ -0,0 +1,70 @@
+//
+//  TileLoDTester.hpp
+//  G3MiOSSDK
+//
+//  Created by Jose Miguel SN on 4/12/15.
+//
+//
+
+#ifndef TileLoDTester_hpp
+#define TileLoDTester_hpp
+
+#include "TileTessellator.hpp"
+#include "TileTexturizer.hpp"
+#include "Context.hpp"
+
+class TileLoDTesterData{
+  //Empty class. Each TileLoDTester will implement a different set of associated data and will
+  //store it inside the tile using its unique level id
+public:
+#ifdef C_CODE
+  virtual ~TileLoDTesterData() { }
+#endif
+#ifdef JAVA_CODE
+  void dispose(){}
+#endif
+};
+
+class TileLoDTester{
+  
+  TileLoDTester* _nextTesterRightLoD;
+  TileLoDTester* _nextTesterWrongLoD;
+  TileLoDTester* _nextTesterVisible;
+  TileLoDTester* _nextTesterNotVisible;
+  
```

Comment:
I agree on the uppercase O. Most literature writes LoD like that, but I agree that we should keep consistent our lowerCamelCase policy.

Regarding the TileLoDTester implementation, the main point of having this super class is to encapsulate the delegation of the response to the next instances. That's something that is done by all subclasses this far.